paris/src/main/java/com/airbnb/paris/typed_array_wrappers/MapTypedArrayWrapper.kt

```diff
@@ -158,6 +165,21 @@ internal class MapTypedArrayWrapper constructor(
         }
     }
 
+    private fun getFontValue(
+        index: Int,
+        resourceGetter: (Int) -> Typeface?,
+        stringValueGetter: (String) -> Typeface?
+    ): Typeface? {
```

Comment:
Doesn't seem like this needs to be a separate function, all this logic could live directly in `getFont(...)`. The reason there's a `getValue(...)` function is it's used several times.